Cargo.toml

```diff
@@ -17,6 +17,7 @@ platform_visionfive2 = []
 
 [profile.dev]
 panic = "abort"
+opt-level = 3
```

Comment:
I am not sure if we should commit to always run with optimization.
What is the current status of Linux? Does it always work with `opt-level = 3`? Is the change to timer frequency necessary?

Reply:
I was thinking about passing the CI by default: we may disable it for dev with other cpu than sifive-u54.

1. It works (linux boots) without changing linux frequency.
2. It doesn't work (linux never boots) with debug mode and the shell eventually freeze and we reach max number of exits. I don't know yet if the frequency change something to this.